src/Quartz.Benchmark/CronExpressionBenchmark.cs

```diff
@@ -0,0 +1,43 @@
+using BenchmarkDotNet.Attributes;
+
+namespace Quartz.Benchmark
+{
+    [MemoryDiagnoser]
+    public class CronExpressionBenchmark
+    {
+        private readonly List<DateTime> dates = new()
+        {
+            new DateTime(2005, 6, 1, 23, 16, 0).ToUniversalTime()
+        };
+
+        private readonly List<string> cronExpressions = new()
+        {
+            "0 15 10 6,15 * ? *",
+            "0 15 10 6,15 * ? *",
+            "0 15 10 15 * ? *",
+            "0 15 10 15,31 * ? *",
+            // Expressions not compatible with 3.x
+            // "0 15 10 6,15,LW * ? 2010",
+            // "0 15 10 6,15,L * ? 2010",
+            // "0 15 10 15,L * ? 2010",
+            // "0 15 10 15,31 * ? 2010",
+            // "0 15 10 15,L-2 * ? 2010",
+            // "0 15 10 31,L-2 * ? 2010",
+            // "0 15 10 1,3,6,15,L * ? 2010",
+            // "0 15 10 15,LW-2 * ? 2010"
+        };
+
+        [Benchmark]
+        public void CreateExpressionsAndCalculateFireTimeAfter()
+        {
+            foreach (var exp in cronExpressions)
```

Comment:
maybe create a `Params` test so that we get results for each expresion? https://benchmarkdotnet.org/articles/overview.html#params

Reply:
Thanks, I'll look into that, the benchmark goal was just to "do the aggregate" and compare results.